### PR TITLE
Backend changes

### PR DESCRIPTION
**Pull request by adam-sas-on**
Update backend changes into frontend to enable loading and changing avatars.
There are some migrations so "manage.py migrate" might be necessary.

### CODE REVIEW

File: onboarding/models.py [hunk 1]
```diff
@@ -18,6 +18,9 @@ def __str__(self):
 
 
 class CompanyQuestionAndAnswer(models.Model):
+    """
+
+    """
```
Comment (kMatejak) [inline onboarding/models.py:23]: it would be worth adding something here :)

File: onboarding/models.py [hunk 2]
```diff
@@ -40,29 +43,24 @@ def create_user(self, email, password, **extra_fields):
...
     now = timezone.now()
     base, extension = os.path.splitext(filename.lower())
     milliseconds = now.microsecond // 1000
     return f"users/{instance.pk}/{now:%Y%m%d%H%M%S}{milliseconds}{extension}"
 
 
 class User(AbstractUser):
+    """
+
+    """
```
Comment (kMatejak) [inline onboarding/models.py:63]: same as in CompanyQuestionAndAnswer

File: onboarding/models.py [hunk 3]
```diff
@@ -83,7 +81,10 @@ def __str__(self):
         return self.email
 
 
-class ContactForm(models.Model):
+class ContactRequestDetail(models.Model):
+    """
+
+    """
```
Comment (kMatejak) [inline onboarding/models.py:87]: same as above

**Comment by kMatejak** [comment]
nice :)

File: onboarding/models.py (hunk 1 above)
Comment (adam-sas-on) [reply onboarding/models.py:23]: True (y)

**Review by annapakula — APPROVED** [review]

File: onboarding/models.py [hunk 4]
```diff
@@ -40,29 +43,24 @@ def create_user(self, email, password, **extra_fields):
...
-        extra_fields.setdefault('is_staff', True)
-        extra_fields.setdefault('is_superuser', True)
-        extra_fields.setdefault('is_active', True)
-
-        if extra_fields.get('is_staff') is not True:
-            raise ValueError(_('Superuser must have is_staff=True.'))
-        if extra_fields.get('is_superuser') is not True:
-            raise ValueError(_('Superuser must have is_superuser=True.'))
-        return self.create_user(email, password, **extra_fields)
-
```
Comment (maciejjankowski) [inline onboarding/models.py:56]: czemu usuwamy?

**Pull request merged**